cmd/kubeadm/app/apis/output/types.go

```diff
@@ -82,3 +82,38 @@ type UpgradePlan struct {
 
 	ConfigVersions []ComponentConfigVersionState
 }
+
+// Certificate represents information for a certificate or a certificate authority when using the check-expiration command.
+type Certificate struct {
+	// Name of the certificate.
+	Name string
+
+	// ExpirationDate defines certificate expiration date.
+	ExpirationDate metav1.Time
+
+	// ResidualTime represents the residual time before expiration.
```

Comment:
```suggestion
	// ResidualTime represents the residual time before expiration.
	// The format of the time is a Go duration string, for example, "24h12m10s".
```
please update this in the private v1alpha3 type too.

this actually looks like
```
  residualTime: 87599h30m15.274289164s
```

i think it's fine and non-go clients can parse it relatively simple.

or should we do like core k8s APIs use `residualTimeSeconds: 123456`?
@pacoxu @carlory what do you think is better.

Reply:
`ResidualTime` is similar to `Age` in kubectl output. 

For json/yaml output, `residualTimeSeconds` would be better.

Reply:
residualTimeSeconds +1